zerver/openapi/zulip.yaml

```diff
@@ -5491,6 +5493,16 @@ paths:
               - type: string
               - type: integer
           example: 43
+        - name: anchor_date
+          in: query
+          description: |
+            Datetime (in ISO 8601 format) to search for the messages closest to this datetime.
```

Comment:
We should add a `**Changes**` entry documenting the API change with a feature level bump; check out the new application feature tutorial for a guide on this process.

I think this should also explicitly state that datetimes that do not include a timezone will be interpreted as UTC... assuming that's the behavior we intend.

Reply:
done